kube/services/netpolicy/networkpolicy_revproxy_templ.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: networkpolicy-revproxy
+  namespace: default
+spec:
+  podSelector:
+    matchLabels:
+      app: revproxy
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            app: revproxy
+      - podSelector:
+          matchLabels:
+             app: portal
+      - ipBlock:
+           cidr: 0.0.0.0/0
+      ports:
+         - port: 80
+         - port: 443
+  egress:
+    - to:
+      - podSelector:
+          matchLabels:
+            app: revproxy
+      - podSelector:
+          matchLabels:
+             app: indexd
+      - podSelector:
+          matchLabels:
+             app: peregrine
+      - podSelector:
+          matchLabels:
+             app: sheepdog
+      - podSelector:
+          matchLabels:
+             app: portal
+      - podSelector:
+          matchLabels:
+             app: fence
+      - ipBlock:
+           cidr: cloudproxy_ip/32
```

Comment:
Would you change this to something like `GEN3_CLOUDPROXY_IP` - so it's more clear that it gets filled in by our template processing, and isn't some kind of kubernetes variable?  Maybe add a comment?  Do the same to the other files?